introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-lst-format.ipynb

```diff
@@ -0,0 +1,733 @@
+    "print(\"Result: label - \" + object_categories[index] + \", probability - \" + str(result[index]))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Clean up\n",
+    "\n",
+    "When we're done with the endpoint, we can just delete it and the backing instances will be released. Uncomment and run the following cell to delete the endpoint and model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "sage.delete_endpoint(EndpointName=endpoint_name)"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
```

Comment:
Can you add the below copyright statement to the notebook metadata?

`"notice": "Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.  Licensed under the Apache License, Version 2.0 (the \"License\"). You may not use this file except in compliance with the License. A copy of the License is located at http://aws.amazon.com/apache2.0/ or in the \"license\" file accompanying this file. This file is distributed on an \"AS IS\" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.",`

Reply:
Added to metadata.